scripts/esxi-esxui_install.sh

```diff
@@ -0,0 +1,5 @@
+# Install ESXi Embedded Host Client directly from VMware website, make
+# sure you have internet connection available or change the URL to a local
+# one.
+esxcli network firewall ruleset set -e true -r httpClient
```

Comment:
This firewall operation is performed by the vmware tools install as well, it's safe to remove it or even better you could refactor this change in a generic "esxi settings" script that runs as first provisioner.